projects/ngu-carousel/src/lib/ngu-carousel/ngu-carousel.component.ts

```diff
@@ -554,6 +562,10 @@ export class NguCarousel<T> extends NguCarouselStore
       dism += `${this.styleid} > .item {transition: transform .6s ease;}`;
     }
 
+    let breakpoints = this.inputs.gridBreakpoints;
+    
+    console.log("breakpoints", breakpoints)
```

Comment:
I'll do another commit and delete this once I get additional feedback. Looks like it slipped in